mne_bids/write.py

```diff
@@ -1279,6 +1301,20 @@ def write_raw_bids(raw, bids_basename, bids_root, events_data=None,
                 _coordsystem_json(raw, unit, orient,
                                   coord_frame, coordsystem_fname, kind,
                                   overwrite, verbose)
+        elif kind == 'eeg':
+            # We only write EEG electrodes.tsv and coordsystem.json
+            # if we have LPA, RPA, and NAS available to rescale to a known
+            # coordinate system frame
+            coords = _extract_landmarks(raw.info['dig'])
+            if set(['RPA', 'NAS', 'LPA']) == set(list(coords.keys())):
+                # Rescale to MNE-Python "head" coord system, which is the
+                # "ElektaNeuromag" system, and equivalent to "CapTrak"
+                pass
```

Comment:
are you going to put the conversion code here?

what happens if NOT `set(['RPA', 'NAS', 'LPA']) == set(list(coords.keys()))`?

Reply:
idk... :p I copied this from your PR. Any recommendation here?

Reply:
okay, so when RPA, NAS and LPA are NOT present, do nothing but write a warning that we can only write EEG coords if we have those.

now for the case that we DO have these coords --> convert them using this handy MNE function:

```
In [1]: import mne                                                              

In [2]: ? mne.channels.montage.transform_to_head                                
Signature:  mne.channels.montage.transform_to_head(montage)
Docstring:
Transform a DigMontage object into head coordinate.

It requires that the LPA, RPA and Nasion fiducial
point are available. It requires that all fiducial
points are in the same coordinate e.g. 'unknown'
and it will convert all the point in this coordinate
system to Neuromag head coordinate system.

Parameters
----------
montage : instance of DigMontage
    The montage.

Returns
-------
montage : instance of DigMontage
    The montage after transforming the points to head
    coordinate system.
File:      ~/miniconda3/envs/sp/lib/python3.6/site-packages/mne/channels/montage.py
Type:      function

```

and then write them ... HOWEVER; please still write a log message that electrode positions are being written and that the user should double check that these are NOT template positions.

Template positions should be removed before writing to BIDS using some kind of `raw.montage.remove()` (don't know the actual method right now)

and then in code above that log message you make a comment `XXX: to be improved, see https://github.com/mne-tools/mne-bids/issues/264`

Reply:
and in any case also add a log message that coords are converted to CapTrak system (i.e., `head`)

Reply:
you can remove the `pass` from line 1364 @adam2392